scripts/integration-tests

```diff
@@ -28,6 +28,21 @@ KUBECTL_VERSION=v1.10.3
 curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl && \
 	chmod +x kubectl && sudo mv kubectl /usr/local/bin/
 
+#
+# In case if a failure, collecting some evidence for further debugging
+#
+function error_collection {
```

Comment:
Love this! One small nit if you re-spin: Maybe put some sort of flashy logging at the start, something like:

```
echo "=====> ERROR: Timed out waiting for NSM daemonset to start"
```

Reply:
done